cypress/e2e/regression/sidebar_2.cy.js

```diff
@@ -30,12 +30,6 @@ describe('Sidebar added sidebar tests', () => {
     sideBar.verifySafeNameExists(newSafeName)
   })
 
-  it('Verify a safe can be removed', () => {
```

Comment:
I would skip it now and update to verify remove for the safes from watchlist

Reply:
Updated